src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -192,8 +198,10 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 		return this._filter;
 	}
 
-	public onSessionStopped(notification: azdata.ProfilerSessionStoppedParams) {
-		this._notificationService.error(nls.localize("profiler.sessionStopped", "XEvent Profiler Session stopped unexpectedly on the server {0}.", this.connection.serverName));
+	public onSessionStopped(notification: azdata.ProfilerSessionStoppedParams, isFileSession: boolean = false) {
```

Comment:
does `isFileSession` need to be passed in here or can `isXELFileSession` be used?

Reply:
Yes, the getter property can be used here. Updated it here